src/SFA.DAS.ApplyService.Web.UnitTests/TaskListViewModelTests.cs

```diff
@@ -628,10 +628,26 @@ public void Whos_in_control_section_status_shows_as_completed_if_companies_house
             model.CharityCommissionDataConfirmed = false;
             model.CharityCommissionManualEntry = true;
             model.WhosInControlConfirmed = false;
+            model.WhosInControlStarted = false;
 
             model.WhosInControlSectionStatus.Should().Be(TaskListSectionStatus.Completed);
         }
 
+        [Test]
+        public void Whos_in_control_section_status_shows_as_pending_if_not_verified_by_companies_house_or_charity_commission_and_whos_in_control_has_started_but_not_confirmed()
+        {
+            var model = GetTaskListViewModelWithSectionsUpToWhosInControlCompleted();
+
+            model.VerifiedCompaniesHouse = false;
+            model.VerifiedCharityCommission = false;
+            model.CompaniesHouseDataConfirmed = false;
+            model.CharityCommissionDataConfirmed = false;
+            model.WhosInControlConfirmed = false;
+            model.WhosInControlStarted = true;
+
```

Comment:
you only need WhoIsInControlStarted = true for the Should to be true.  Either take out the rest, or put in test cases where they all vary to check the Should() isn't affected.